test/base_test_fixture.h

```diff
@@ -1502,6 +1502,39 @@ struct base_test_fixture
         results.get_ref(0, ref);
         REQUIRE(ref == name);
     }
+    void string_vector_test()
+    {
+        nanodbc::connection connection = connect();
+        REQUIRE(connection.native_dbc_handle() != nullptr);
+        REQUIRE(connection.native_env_handle() != nullptr);
+        REQUIRE(connection.transactions() == std::size_t(0));
+
+        const std::vector<nanodbc::string_type> first_name = {NANODBC_TEXT("Fred"), NANODBC_TEXT("Barney"), NANODBC_TEXT("Dino")};
+        const std::vector<nanodbc::string_type> last_name = {NANODBC_TEXT("Flintstone"), NANODBC_TEXT("Rubble"), NANODBC_TEXT("")};
+
+        drop_table(connection, NANODBC_TEXT("string_vector_test"));
+        execute(connection, NANODBC_TEXT("create table string_vector_test (first varchar(10), last varchar(10));"));
+
+        nanodbc::statement query(connection);
+        prepare(query, NANODBC_TEXT("insert into string_vector_test(first, last) values(?, ?)"));
+        query.bind_strings(0, first_name);
+
+        // With null sentry
+        query.bind_strings(1, last_name, NANODBC_TEXT(""));
+        nanodbc::execute(query);
+
+        nanodbc::result results = execute(connection, NANODBC_TEXT("select first,last from string_vector_test order by first desc;"));
+        for (auto &vals : results) {
```

Comment:
I don't think this look is right.

Caught by [latest AppVeyor build](https://ci.appveyor.com/project/lexicalunit/nanodbc/build/1.0.321/job/ljosm8j6spte5bji) as ` warning C4189: 'vals': local variable is initialized but not referenced`